dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -445,18 +445,27 @@ public void PaneOpenForceCloseTest()
 
         [TestMethod]
         [TestProperty("TestSuite", "A")]
-        public void PaneClosedWhenAutoAndNarrow()
+        public void PaneClosedUponLaunch()
         {
             var testScenarios = RegressionTestScenario.BuildLeftNavRegressionTestScenarios();
             foreach (var testScenario in testScenarios)
             {
                 using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "Navigation Minimal Test" }))
                 {
+                    Log.Comment("Verify that NavigationView with DisplayMode set to 'Auto' and a narrow width does not display pane on load.");
+                    CheckBox isAutoPaneOpenCheckBox = new CheckBox(FindElement.ById("IsAutoPaneOpenCheckBox"));
+                    Wait.ForIdle();
+                    Verify.IsTrue(isAutoPaneOpenCheckBox.ToggleState == ToggleState.Off);
+
+                    Log.Comment("Verify that NavigationView with DisplayMode set to 'LeftMinimal' does not display pane on load.");
+                    CheckBox isLeftMinimalPaneOpenCheckBox = new CheckBox(FindElement.ById("IsLeftMinimalPaneOpenCheckBox"));
+                    Wait.ForIdle();
```

Comment:
Since the state of the app is not changing, there is no need for the multiple WaitForIdle calls. Just one should suffice.